frontend/src/pages/org/collection-editor.ts

```diff
@@ -99,6 +99,9 @@ export class CollectionEditor extends LiteElement {
   @property({ type: String })
   orgId!: string;
 
+  @property({ type: Boolean })
+  orgStorageQuotaReached!: boolean;
```

Comment:
Prefer to set a default boolean value if the app can still function (versus something like an ID string, in which there is no default value without breaking functionality:)
```suggestion
  orgStorageQuotaReached = false;
```